egs2/swbd/asr1/conf/train_asr.yaml

```diff
@@ -1 +0,0 @@
-tuning/train_asr_conformer2.yaml
```

Comment:
Maybe, we can remove this config?

Reply:
Or add a pointer to your new conformer config.

Reply:
I added a pointer to the new conformer config.